pyproject.toml

```diff
@@ -141,8 +143,10 @@ gdown = "*"
 matplotlib = "*"
 seaborn = "*"
 tabulate = "*"
+browsergym = "0.10.2"
```

Comment:
```suggestion
```

I think we're trying to avoid importing the entirety of browsergym and only importing the parts that we need.